appcli/app_test.go

```diff
@@ -263,7 +264,7 @@ func TestGetConfigMetadata_quiet(t *testing.T) {
 		)
 	}
 
-	if !metadata.Quiet {
+	if metadata.Verbosity != ui.VerbosityLevelQuiet {
```

Comment:
Mind adding a test for `ui.VerbosityLevelSilent`? 

It would look pretty much exactly the same as this one, but for a silent flag.